src/librustc_data_structures/bitvec.rs

```diff
@@ -139,14 +133,14 @@ impl FromIterator<bool> for BitVector {
 /// one gigantic bitvector. In other words, it is as if you have
 /// `rows` bitvectors, each of length `columns`.
 #[derive(Clone)]
-pub struct BitMatrix {
+pub(crate) struct BitMatrix {
```

Comment:
I'm of two minds. I think I would keep this `pub`, since it's meant to be reused, but on the other hand I like keeping the privacy of things *tight* as a way to document where they are *actually* used. I guess `pub(crate)` is ok.